scheduler/http-basic-container-config.edn

```diff
@@ -0,0 +1,38 @@
+{:port #config/env-int "COOK_PORT"
+ :authorization {:http-basic true}
+ :authorization-config {;; What function should be used to perform user authorization?
+                        ;; See the docstring in cook.authorization for details.
+                        :authorization-fn cook.authorization/configfile-admins-auth-open-gets
+                        ;; These users have admin privileges when using
+                        ;; configfile-admins-auth -- that is, they can view and modify other
+                        ;; users' jobs.
+                        :admins #{"root"}
+                        ;; users that are allowed to do things on behalf of others
+                        :authorized-impersonators #{"root" "poser"}}
+ :database {:datomic-uri #config/env "COOK_DATOMIC_URI"}
+ :hostname #config/env "COOK_HOSTNAME"
+ :zookeeper {:local? #config/env-bool "COOK_ZOOKEEPER_LOCAL"
+             :connection #config/env "COOK_ZOOKEEPER"}
+ :scheduler {:offer-incubate-ms 15000
+             :task-constraints {:timeout-hours 1
+                                :timeout-interval-minutes 1
+                                :memory-gb 48
+                                :retry-limit 15
+                                :cpus 6}}
+ :agent-query-cache {:ttl-ms 1000}
+ :rate-limit {:user-limit-per-m 1000000}
+ :rebalancer {:dru-scale 1}
+ :mesos {:master #config/env "MESOS_MASTER"
+         :master-hosts [#config/env "MESOS_MASTER_HOST"]
+         :failover-timeout-ms nil ; When we close the instance of Cook, all its tasks are killed by Mesos
+         :leader-path "/cook-scheduler"
+         :framework-id #config/env "COOK_FRAMEWORK_ID"}
+ :unhandled-exceptions {:log-level :error}
+ :metrics {:jmx true}
+ :nrepl {:enabled? true
+         :port #config/env-int "COOK_NREPL_PORT"}
+ :log {:file #config/env "COOK_LOG_FILE"
+       :levels {"datomic.db" :warn
+                "datomic.peer" :warn
+                "datomic.kv-cluster" :warn
+                :default :info}}}
```

Comment:
I combined the config files used by Travis, but I'd prefer not merging these because they're used inside a docker container. I tried that out locally, and I had to modify `run-docker.sh` to pass additional environment variables from the host through to the container. I'd rather not have to worry about correctly setting those environment variables when I'm testing. We have other sample config files in the scheduler folder, so I don't think one more will hurt.

Reply:
Instead of adding another config, would it make sense to have multiple "run docker" scripts, e.g. `run-docker-basic-auth.sh`?